packages/https-proxy/lib/ca.coffee

```diff
@@ -176,7 +176,7 @@ class CA
     hosts = [].concat(hosts)
 
     mainHost   = hosts[0]
-    keysServer = pki.rsa.generateKeyPair(1024)
+    keysServer = pki.rsa.generateKeyPair(2048)
```

Comment:
get a rough idea of time change here

Reply:
added some `console.time` statements to `ca_spec`, here are the measurements for different key sizes:

cert + ca bits | time spent creating CA | time spent generating 1 domain keypair
---- | ---- | ----
512 | 13.967ms | 14.760ms
1024 | 21.039ms | 38.133ms
2048 | 82.628ms | 70.606ms

Reply:
Can you also output / catalog the timings when run from a docker container?

Maybe just add the debug logs in a circle ci build so I have a reference of the differences specifically for CI.

Reply:
alright, i committed the `console.time` logs to ca_spec